contracts/interfaces/IStableDebtToken.sol

```diff
@@ -14,17 +14,17 @@ interface IStableDebtToken is IInitializableDebtToken {
    * @dev Emitted when new stable debt is minted
    * @param user The address of the user who triggered the minting
    * @param onBehalfOf The recipient of stable debt tokens
-   * @param amount The amount minted (user entered amount + balance increase from interest)
-   * @param currentBalance The current balance of the user
-   * @param balanceIncrease The increase in balance since the last action of the user
+   * @param value The amount minted (user entered amount + balance increase from interest)
+   * @param currentBalance The balance of the user based on the previous balance and balance increase from interest
+   * @param balanceIncrease The increase in balance since the last action of the user 'onBehalfOf'
    * @param newRate The rate of the debt after the minting
    * @param avgStableRate The next average stable rate after the minting
    * @param newTotalSupply The next total supply of the stable debt token after the action
    **/
   event Mint(
     address indexed user,
     address indexed onBehalfOf,
-    uint256 amount,
+    uint256 value,
```

Comment:
Changed the name of this parameter (for uniformity). Open to revert it back if needed (may break previous event integrations)